x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -137,16 +140,82 @@ export const getActionStatus = (status: ResponseActionStatus): string => {
   return '';
 };
 
+// type guards to ensure only the matching string values are attached to the filter type
+export const isAgentType = (type: string): type is typeof RESPONSE_ACTION_AGENT_TYPE[number] =>
+  RESPONSE_ACTION_AGENT_TYPE.includes(type as typeof RESPONSE_ACTION_AGENT_TYPE[number]);
+
+export const isActionType = (type: string): type is typeof RESPONSE_ACTION_TYPE[number] =>
+  RESPONSE_ACTION_TYPE.includes(type as typeof RESPONSE_ACTION_TYPE[number]);
+
```

Comment:
these are awesome. I se you are exporting them out of this `hooks` folder. Should they maybe go into `x-pack/plugins/security_solution/common/endpoint/service/response_actions/type_guards.ts` instead?

Reply:
done 683d4e12caf080605420c6c16147b9ed82abb0b3